common/fw-update-helper.cpp

```diff
@@ -191,7 +191,8 @@ namespace rs2
         // if device is D457, and fw is signed - using mipi specific procedure
```

Comment:
Please update comment to all MIPI devices and not only D457

Reply:
fixed.